cln-rpc/src/model.rs

```diff
@@ -0,0 +1,338 @@
+	    pub unilateraltimeout: Option<u32>,
+	    #[serde(alias = "destination", skip_serializing_if = "Option::is_none")]
+	    pub destination: Option<String>,
+	    #[serde(alias = "fee_negotiation_step", skip_serializing_if = "Option::is_none")]
+	    pub fee_negotiation_step: Option<String>,
+	    #[serde(alias = "wrong_funding", skip_serializing_if = "Option::is_none")]
+	    pub wrong_funding: Option<String>,
+	    #[serde(alias = "force_lease_closed", skip_serializing_if = "Option::is_none")]
+	    pub force_lease_closed: Option<bool>,
+	}
+
+}
+
+
+pub mod responses {
+    #[allow(unused_imports)]
+    use crate::primitives::*;
+    #[allow(unused_imports)]
+    use serde::{{Deserialize, Serialize}};
+
+	/// Type of connection
+	#[derive(Copy, Clone, Debug, Deserialize, Serialize)]
+	#[serde(rename_all = "lowercase")]
+	pub enum GetinfoAddressType {
+	    DNS,
+	    IPV4,
+	    IPV6,
+	    TORV2,
+	    TORV3,
+	    WEBSOCKET,
+	}
+
+	#[derive(Clone, Debug, Deserialize, Serialize)]
+	pub struct GetinfoAddress {
+	    // Path `Getinfo.address[].type`
+	    #[serde(rename = "type")]
+	    pub item_type: GetinfoAddressType,
+	    #[serde(alias = "port")]
+	    pub port: u16,
+	    #[serde(alias = "address", skip_serializing_if = "Option::is_none")]
+	    pub address: Option<String>,
+	}
+
+	/// Type of connection
+	#[derive(Copy, Clone, Debug, Deserialize, Serialize)]
+	#[serde(rename_all = "lowercase")]
+	pub enum GetinfoBindingType {
+	    LOCAL_SOCKET,
+	    IPV4,
+	    IPV6,
+	    TORV2,
+	    TORV3,
+	}
+
+	#[derive(Clone, Debug, Deserialize, Serialize)]
+	pub struct GetinfoBinding {
+	    // Path `Getinfo.binding[].type`
+	    #[serde(rename = "type")]
+	    pub item_type: GetinfoBindingType,
+	    #[serde(alias = "address", skip_serializing_if = "Option::is_none")]
+	    pub address: Option<String>,
+	    #[serde(alias = "port", skip_serializing_if = "Option::is_none")]
+	    pub port: Option<u16>,
+	    #[serde(alias = "socket", skip_serializing_if = "Option::is_none")]
+	    pub socket: Option<String>,
+	}
+
+	#[derive(Clone, Debug, Deserialize, Serialize)]
+	pub struct GetinfoResponse {
+	    #[serde(alias = "id")]
+	    pub id: String,
+	    #[serde(alias = "alias")]
+	    pub alias: String,
+	    #[serde(alias = "color")]
+	    pub color: String,
+	    #[serde(alias = "num_peers")]
+	    pub num_peers: u32,
+	    #[serde(alias = "num_pending_channels")]
+	    pub num_pending_channels: u32,
+	    #[serde(alias = "num_active_channels")]
+	    pub num_active_channels: u32,
+	    #[serde(alias = "num_inactive_channels")]
+	    pub num_inactive_channels: u32,
+	    #[serde(alias = "version")]
+	    pub version: String,
+	    #[serde(alias = "lightning-dir")]
+	    pub lightning_dir: String,
+	    #[serde(alias = "blockheight")]
+	    pub blockheight: u32,
+	    #[serde(alias = "network")]
+	    pub network: String,
+	    #[serde(alias = "fees_collected_msat")]
+	    pub fees_collected_msat: Amount,
+	    #[serde(alias = "address[]")]
```

Comment:
```suggestion
	    #[serde(alias = "address")]
```

The alias should be the name of the field, so the name should not contain [] ? or I'm missing something?